tests/baseTest.py

```diff
@@ -2,76 +2,88 @@
 import unittest
 from app import create_app
 import json
-from app.models import dbconnect, droptables
+from app.models import dbconnect, droptables, createTables
 
 class BaseTestClient(unittest.TestCase):
     '''the base test configurations for the entire application'''
-    def __init__(self):
-        pass
 
     def setUp(self):
         '''creates the data required in the tests. also connects to the test database where the data will be stored'''
-        dbconnect()
         self.app = create_app('testing').test_client()
+        test = dbconnect()
+        createTables()
 
-    user_information = {
-        "username":"Spongebob",
-        "email": "spongebobsquarepants@bikinibottom.sea",
-        "password": "CrustyKr1abs"
-    }
-
-    test_valid_product = {
-        "name" : "hp",
-        "description" : "elite",
-        "category" : "computers",
-        "quantity" : 10,
-        "price" : 50000
+        self.user_information = {
```

Comment:
Remove these testing data and urls from the setup functions, you can import them from a different file

Reply:
Can it be a part of a utils file? im not quite sure about implementing this